### DISCUSSION TITLE
Youtube problem

### DISCUSSION THREAD

**Discussion by knglerxst** [streamlink/streamlink#3603] [General]
Streamlink is behaving differently with live Youtube stream links. It used to be able to open streams within several seconds and play them reliably. Now, it takes minutes to open VLC after I give it a link, and the playback is choppy half of the time or it drops the connection midstream.  This is the command format I use: streamlink youtube.com/watch?v=[alphanumeric string] 360p. I noticed it works better with .m3u8 urls on other sites like D-Live and Twitch. I'm using the latest stable release. Has anyone else noticed this, or does anyone know what might be causing the problem?

**Comment by WolfganP** [comment]
As in any troubleshooting issue, make sure you're using the latest streamlink version and use the `--loglevel debug` parameter, then post the results including the full command line. Easier for helping you.
